_testenv.yml

```diff
@@ -0,0 +1,14 @@
+default:
```

Comment:
This file will be used as a template to compose `testenv.yml`.
The placeholders in this file will be replaced with values from vault.
If we need to extend the config, we can add new keys to this file and add new values in vault.